composer/workflows/airflow_db_cleanup.py

```diff
@@ -79,9 +83,10 @@
 DAG_OWNER_NAME = "operations"
 # List of email address to send email alerts to if this job fails
 ALERT_EMAIL_ADDRESSES = []
+# Airflow version used by the environment
+AIRFLOW_VERSION = airflow.version.version.split(".")
```

Comment:
AIRFLOW_VERSION will be something like this ["2", "1", "2+composer"] which is not what would reader expect and can mess up with list comparisons below, right?

Reply:
You are right, fixed.